pkg/extkingpin/flags.go

```diff
@@ -68,6 +68,18 @@ func RegisterCommonTracingFlags(app FlagClause) *extflag.PathOrContent {
 	)
 }
 
+// RegisterRequestLoggingFlags registers flags to pass a request logging configuration to be used.
+func RegisterRequestLoggingFlags(app FlagClause) *extflag.PathOrContent {
+	return extflag.RegisterPathOrContent(
+		app,
+		// TODO @yashrsharma44: Discuss the name for this flag :).
```

Comment:
`request.logging` or something else? :cat:

Reply:
Makes sense to me (:

Reply:
Cool :grin: